eml_parser/indicators.py

```diff
@@ -40,3 +45,28 @@ def __hash__(self):
                 self.sha256,
             )
         )
+
+    @staticmethod
+    def parse_auth_results_header(headers: list) -> dict:
+        value = ""
+        auth_types = {}
+
+        if headers:
+            for header in headers:
+                name = header.get("name")
+                if name == "Authentication-Results":
```

Comment:
Maybe all this text like "Authentication-Results", "dkim" and so one put into constant file it will be easy to change